bigquery/google/cloud/bigquery/dataset.py

```diff
@@ -26,6 +27,14 @@
 from google.cloud.bigquery.table import TableReference
 
 
+_W_PREFIX = re.compile(
+    r"""
+    (\S*)\:(?P<ref>\S*)
```

Comment:
Since at least one character is required, this should probably be `\S+`, right?

Also, `ref` isn't all that meaningful to me. How about `remaining`, since it's everything after the `:` character?